trap.go

```diff
@@ -188,6 +188,26 @@ func (t *TrapListener) Listen(addr string) error {
 			if traps != nil {
 				t.OnNewTrap(traps, remote)
 			}
+			if traps.PDUType == InformRequest {
+				traps.PDUType = GetResponse
+				log.Printf("sending INFORM response: %+v\n", traps)
+
+				ob, err := traps.marshalMsg()
+				if err != nil {
+					return fmt.Errorf("Error marshaling INFORM response: %v\n", err)
```

Comment:
Remove newline on error types.  I usually don't add them on log messages either.

Reply:
Right, good catch. I think this had started as a log message, and I'm used to a different log package which doesn't add the newlines when doing formatted output (same as `fmt.Printf`). Will correct.

Reply:
OK, so I'm removing the error newlines, as they should be, but I'm leaving them on the logs to be consistent with the rest of the package.